docker/test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+### Test script pulling the docker image and use it against showcase proto.
+
+# Docker image tag: gapic-generator-typescript:latest.
+# Create new directory showcase-typescript. 
+mkdir showcase-typescript
```

Comment:
It will fail if it already exists. Let's do `rm -rf .showcase-typescript` first, then `mkdir`. Also, it makes sense to put its name into a variable since it's used more than once.

Reply:
Also, I think the dot is missing (should be `.showcase-typescript`, as it's in `.gitignore`).

Reply:
Thanks!

Reply:
`showcase-typescript` occurs three times in this script, let's have a variable `OUTPUT_DIR=.showcase_typescript`. That way it would be much easier to change the location if needed.  Also, please note that the `.gitignore` says `.showcase-typescript`  (with a dot).

Reply:
Actually, it just looks you didn't push your latest changes :)

Reply:
sorry!